tests/api/api.spec.ts

```diff
@@ -696,6 +696,48 @@ suite("@dataform/api", () => {
         );
       });
     });
+
+    test("contextual pre and post ops", async () => {
```

Comment:
i would suggest moving this `test()` call *inside* the foreach loop. that way you can include the `warehouse` name as part of the test name, and you'll have singular tests for each warehouse.